projects/vdk-plugins/vdk-plugin-control-cli/tests/test_secrets_plugin.py

```diff
@@ -61,14 +61,15 @@ def test_secrets_plugin_no_url_configured():
     ):
         runner = CliEntryBasedTestRunner(vdk_plugin_control_cli, secrets_plugin)
 
-        result: Result = runner.invoke(
-            ["run", jobs_path_from_caller_directory("simple-job")]
-        )
-        # job that do not use secrets should succeed
-        cli_assert_equal(0, result)
+        # TODO: Uncomment this after vdk-core release
+        # result: Result = runner.invoke(
+        #     ["run", jobs_path_from_caller_directory("simple-job")]
+        # )
+        # # job that doesn't explicitly use secrets should fail due to secrets overrides in core
```

Comment:
I think job that do not use secrets should succeed still. 

Simply it won't fetch any configuration from secrets.

Reply:
The test checks what happens when you set `VDK_SECRETS_DEFAULT_TYPE = control-service` but do not provide a url. If you don't set the secrets default type, the job will succeed. The reason it fails is that we no call get_all_secrets() on each job run, so if secrets are misconfigured somehow, it will fail.